Source/Core/Core/Core.cpp

```diff
@@ -240,9 +258,38 @@ void Stop()  // - Hammertime!
 	}
 }
 
+void DeclareAsCPUThread()
+{
+	#ifdef ThreadLocalStorage
+		tls_is_cpu_thread = true;
+	#else
+		//Use pthread implementation for Android
+		//Only initialize pthread keys once
+		#ifndef PTHREAD_IS_CPU_THREAD 
+			#define PTHREAD_IS_CPU_THREAD 
+			pthread_key_create(&s_tls_is_cpu_key, nullptr);
+		#endif
+
```

Comment:
Using the preprocessor makes it *compile* once. You probably want a `static` variable.